timvt/endpoints/factory.py

```diff
@@ -5,6 +5,7 @@
 
 from buildpg.asyncpg import BuildPgPool
 from morecantile import TileMatrixSet
+from shapely import wkt
```

Comment:
ref https://github.com/developmentseed/timvt/pull/48/files#r585808957, shapely isn't part of the requirements and I'd love to avoid adding more requirements for small feature like WKT parsing!